README.md

```diff
@@ -5,11 +5,13 @@ An implementation of [UTP] for Rust using non-blocking sockets.
 
 |Crate|Documentation|Linux/OS X|Windows|Issues|
 |:---:|:-----------:|:--------:|:-----:|:----:|
-|[![](http://meritbadge.herokuapp.com/tokio-utp)](https://crates.io/crates/tokio-utp)|[![Documentation](https://docs.rs/tokio-utp/badge.svg)](https://docs.rs/tokio-utp)|[![Build Status](https://travis-ci.org/maidsafe/tokio-utp.svg?branch=master)](https://travis-ci.org/maidsafe/tokio-utp)|[![Build status](https://ci.appveyor.com/api/projects/status/wk7vqat3xhly1ny1/branch/master?svg=true)](https://ci.appveyor.com/project/MaidSafe-QA/tokio-utp/branch/master)|[![Stories in Ready](https://badge.waffle.io/maidsafe/tokio-utp.png?label=ready&title=Ready)](https://waffle.io/maidsafe/tokio-utp)|
+|[![](http://meritbadge.herokuapp.com/tokio-utp)](https://crates.io/crates/tokio-utp)|[![Documentation](https://docs.rs/tokio-utp/badge.svg)](https://docs.rs/tokio-utp)|[![Build Status](https://travis-ci.org/maidsafe/tokio_utp.svg?branch=master)](https://travis-ci.org/maidsafe/tokio_utp)|[![Build status](https://ci.appveyor.com/api/projects/status/wk7vqat3xhly1ny1/branch/master?svg=true)](https://ci.appveyor.com/project/MaidSafe-QA/tokio-utp/branch/master)|[![Stories in Ready](https://badge.waffle.io/maidsafe/tokio_utp.png?label=ready&title=Ready)](https://waffle.io/maidsafe/tokio_utp)|
```

Comment:
I think the crates.io links may remain valid after the renaming, but not so sure about the others.  To be safe (and consistent with our other projects), it'd be better to change all instances of `tokio-utp` to `tokio_utp` except the AppVeyor link.

Reply:
Changed `name =` back to tokio-utp so that should resolve this issue (no crate renaming will occur). Means inconsistency within this file as crate, docs and Appveyor links will still be `tokio-utp`, while the rest will be `tokio_utp`.